.github/workflows/perf-regression.yml

```
@@ -0,0 +1,190 @@
+            const seoPass = seoDiff >= -5.0 && lhSEO >= 90.0;
+            const k6P95Pass = k6P95Diff <= 5.0 && k6P95 <= 500.0;
+            const errorRatePass = k6ErrorRate <= 1.0;
+
+            const failed = !perfPass || !accPass || !bpPass || !seoPass || !k6P95Pass || !errorRatePass;
+
+            const fmtScore = (num) => num.toFixed(1);
+            const fmtDiff = (num) => (num > 0 ? `+${num.toFixed(1)}%` : `${num.toFixed(1)}%`);
+
+            const commentBody = `### ⚡ Performance Regression Report
+
+| Metric | Baseline (main) | Current PR | Diff | Status |
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)